### ISSUE TITLE
Wheelchair base profile - avoid stairs and pictograms

### ISSUE THREAD

**Issue by Aiurek** [osmandapp/Osmand#9415]
The wheelchair-info is pretty thoroughly mapped in a lot of areas. Here, OsmAnd could be a nice tool for wheelchair users to get appropriate routing e.g. avoid stairs, which might be realized by offering a wheelchair base profile.

Would be nice if there's a base profile for wheelchair users, that:
- includes a pre-setting for avoiding stairs (already implemented in pedestrian roting within #4236)
- includes  a pre-setting to switch on pictograms for wheelchair access and wheelchair access to toilets for a poi (#7080)

So far a wheelchair pictogram is even missing in the new profile generation:

![Wheelchair (2)](https://user-images.githubusercontent.com/27865200/87022382-b3af6580-c1d6-11ea-85e9-198d35ea46ac.jpg)

**Comment by scaidermern** [comment]
This would be a really nice improvement. Information about wheelchair accessibility is one of OSM's strength compared to competitors.

**Comment by dmpr0** [comment]
I have added wheelchair icons for the profile, it will available in the current nightly builds.

**Comment by AndreazLattmann** [comment]
I agree, which is why I intend to propose the inclusion of the tag wheelchair=offroad or similar in the OSM ml tagging.  
Since the paths dedicated to disabled people are increasing.

**Comment by vshcherb** [comment]
Link from #9768 - https://github.com/LorenzoStucchi/Wheelchair-OsmAnd - created profile. Please feel free to test and leave feedback here

**Comment by LorenzoStucchi** [comment]
Thanks, @vshcherb! I'm also available for any improvement that it is think needed. 

I have a question, I had to make a double request to the user for the insertion of a parameter of a maximum inclination since the value could be positive or negative. It is possible to make the user chose only one parameter and after transform it negative?

**Comment by LorenzoStucchi** [comment]
@Aiurek I also have another suggestion, I think that the icon for the wheelchair should be the one of ["The accessible Icon Project"](http://accessibleicon.org/), that has a deep meaning and it is available with CC0 licence.

**Comment by Aiurek** [comment]
@LorenzoStucchi yes, I like the one you suggested even more than the one i suggested. Thx for contributing the routing profile. I think the whole implementation of wheelchair profiles/routing is a really valuable contribution to OsmAnd

**Comment by LorenzoStucchi** [comment]
Step for the presence of routing for wheelchair:

- [x] presence of a wheelchair icon
- [x] presence of kerbs inside the .obf file
- [ ] creation of a wheelchair routing ( see https://github.com/LorenzoStucchi/Wheelchair-OsmAnd )
- [ ] style for the visualization of the kerbs

every suggestion in the creation of the routing is welcome
